doc/user/installing-llvm.md

```diff
@@ -31,6 +35,8 @@ apt-get install clang llvm libc++abi-dev
 
 ## Fedora
 
+The tested version of LLVM for Fedora is 4.0.1.
```

Comment:
Did you check in the Dockerfile? I think it's 3.9.1 on my Fedora 26 system, I'll check tonight.

Reply:
Yes I think I checked in the `Dockerfile`. Please check locally, but the `Dockerfile` configuration is the authoritative one so I'll check there again as well.